Services/Storage/StorageRecords.cs

```diff
@@ -18,6 +18,7 @@ public interface IStorageRecords
     {
         IStorageRecords Init(StorageConfig config);
         Task<StorageRecord> GetAsync(string id);
+        Task<IEnumerable<StorageRecord>> GetAsync(string sqlCondition, SqlParameter[] parameters);
```

Comment:
this couples the interface with Cosmos DB. What's the plan if we want to store data in a different storage type, e.g. Cassandra or Azure Tables etc. ?

Reply:
This class contains all cosmosdb specific requests. I added this so we could specify the query condition.

Reply:
see my last comment, StorageRecords is the only class coupled with Cosmos DB, this couples also the clients of this class